draft-ietf-dnsop-structured-dns-error.md

```diff
@@ -417,6 +416,53 @@ via {{?I-D.reddy-add-resolver-info}} and the client includes the EDE OPT
 pseudo-RR in the query. This allows the server to differentiate
 EDE-aware clients from EDE-unaware clients and respond appropriately.
 
+# New Sub-Error Codes Definition
+
+Subsequent documents defining new Sub-Error codes MUST be accepted by
```

Comment:
This text conflicts with the policy set for the registry (expert review).  I think the intent here is not about the document that requests the new code but the reference that is provided to characterize the error. I agree that it is preferable  to 
 have an IETF ref, but MUST is too strong here.